tests/lib/rules/no-unused-prop-types.js

```diff
@@ -3215,6 +3148,22 @@ ruleTester.run('no-unused-prop-types', rule, {
         line: 11,
         column: 8
       }]
+    }, { // None of the props are used issue #1162
+      code: [
+        'import React from "react"; ',
+        'var Hello = React.createReactClass({',
+        ' propTypes: {',
+        '   name: React.PropTypes.string',
+        ' },',
+        ' render: function() {',
+        '   return <div>Hello Bob</div>;',
+        '  }',
+        '});'
+      ].join('\n'),
+      parser: 'babel-eslint',
```

Comment:
similarly here, what in this test needs babel-eslint? this test seems like it'd work with ES6.